WooCommerce/Classes/ViewRelated/Orders/Order Details/Receipts/ReceiptViewController.swift

```diff
@@ -51,15 +51,33 @@ final class ReceiptViewController: UIViewController {
     }
 
     @objc private func shareReceipt() {
+        ServiceLocator.analytics.track(event: .InPersonPayments.receiptEmailTapped(countryCode: nil,
```

Comment:
I see this is event that was here from before, but would maybe "receiptShareTapped" be a better name? Since we trigger share sheet and users can share it various ways/apps?